example/src/PrintStatePage/PrintPage.tsx

```diff
@@ -8,7 +8,7 @@ export const PrintPage = () => {
     const styles = StyleSheet.create({
         container: {
             alignItems: 'center',
-            marginTop: theme.sw.spacing.m,
```

Comment:
Could you put all those modifications not related to default background color in a separate commit please ?

Reply:
yes, i will make another PR, to make the clean history of commit